accounts/abi/unpack_test.go

```diff
@@ -462,6 +472,35 @@ func TestMultiReturnWithArray(t *testing.T) {
 	}
 }
 
+func TestMultiReturnWithStringArray(t *testing.T) {
+	const definition = `[{"name" : "multi", "outputs": [{"name": "","type": "uint256[3]"},{"name": "","type": "address"},{"name": "","type": "string[2]"},{"name": "","type": "bool"}]}]`
```

Comment:
Just to be sure, I would like to also see a `uint256[]` array _after_ the string slice.